packages/console/src/components/Transfer/index.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from 'classnames';
+import React, { ReactNode, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import CircleMinus from '@/icons/CircleMinus';
+import CirclePlus from '@/icons/CirclePlus';
+import Dragable from '@/icons/Dragable';
+
+import IconButton from '../IconButton';
+import DragDropProvider from './DragDropProvider';
+import DraggableItem from './DraggableItem';
+import * as styles from './index.module.scss';
+
+type TransferItem = {
+  title: ReactNode;
+  value: string;
+};
+
+type Props = {
+  title: string;
+  datasource: TransferItem[];
+  value?: string[];
+  footer?: ReactNode;
+  onChange?: (value: string[]) => void;
```

Comment:
why `onChange` optional?

Reply:
Is a common practice, an example: we only need this component to display things (preview or readonly).

Reply:
nit: it seems that `onChange` is required for the `Transfer` component in most cases. I'm good with both for now.

Reply:
`Transfer` is a standard form component, `onChange` should be optional.